examples/android/strictmode/app/build.gradle

```diff
@@ -53,7 +53,6 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
 
     // You need to build grpc-java to obtain these libraries below.
-    implementation 'io.grpc:grpc-core:1.59.0-SNAPSHOT' // CURRENT_GRPC_VERSION
```

Comment:
This looks wrong. Nothing in this change should impact the dependencies needed by the examples. When we roll forward, we should not make this change and the change to proguard-rules.pro. The change to ManagedChannelImplTest.java is also suspicious.

(This sort of change was hard to spot in #10528 because the commits were broken and bringing in changes that already happened no master (e.g., README.md for the new release, java_grpc_library.bzl).)